opentelemetry-collector-rhde-config.spec

```diff
@@ -7,6 +7,7 @@ BuildArch: noarch
 License: Apache-2.0
 Source1: config.yaml
 Source2: opentelemetry-collector-rhde-config.service
+Source3: etc/microshift/manifests/opentelemetry-gateway.yaml
```

Comment:
Do we need to host this file in ` etc/microshift/manifests` in this repo? If no it is simpler to have just `microshift` directory

Reply:
y, we can do  :+1: